docker-storage-setup.1

```diff
@@ -9,6 +9,9 @@ docker\-storage\-setup - Grows the root filesystem and sets up storage for docke
 .PP
 \f[B]-h, --help\f[]
   Print usage statement
+\f[B]--reset\f[]
+  Reset your docker storage to init state.  This command is not sufficient to cleanup your
+  docker environment.  It is used by other tools like \f[B]atomic storage setup\f[]
```

Comment:
Can we mention that reset does not try to remove volume groups or try to remove any of the disks added previously.